src/fluidplayer.js

```diff
@@ -226,7 +226,7 @@ const fluidPlayerClass = function () {
                 },
                 controlForwardBackward: {
                     show: false,
-                    doubleTapMobile: true
+                    doubleTapMobile: false
```

Comment:
This will not fix the issue. The user can still set this back to true and we have the same problem. Plus we want this to be the double tap on mobile to be default.
The problem is in this if: https://github.com/fluid-player/fluid-player/blob/master/src/fluidplayer.js#L1914

Reply:
Updated